cuda_core/tests/test_event.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2021-2024 NVIDIA Corporation.  All rights reserved.
+#
+# Please refer to the NVIDIA end user license agreement (EULA) associated
+# with this source code for terms and conditions that govern your use of
+# this software. Any use, reproduction, disclosure, or distribution of
+# this software and related documentation outside the terms of the EULA
+# is strictly prohibited.
+
+from cuda import cuda
+from cuda.core.experimental._event import  EventOptions, Event
+from cuda.core.experimental._utils import handle_return
+from cuda.core.experimental._device import Device
+import pytest
+
+@pytest.fixture(scope='module')
+def init_cuda():
+    Device().set_current()
+
+def test_is_timing_disabled():
+    options = EventOptions(enable_timing=False)
+    event = Event._init(options)
+    assert event.is_timing_disabled == True
+
+def test_is_sync_busy_waited():
+    options = EventOptions(busy_waited_sync=True)
+    event = Event._init(options)
+    assert event.is_sync_busy_waited == True
+
+def test_is_ipc_supported():
+    options = EventOptions(support_ipc=True)
+    try:
+        event = Event._init(options)
+    except NotImplementedError:
+        assert True

```

Comment:
🤦 my bad, could you create an issue to track this? We should add IPC support in the next release. Then we'll update this test.

(No need to sweat now other than creating a tracking issue!)

Reply:
This was already tracked in #103.

Reply:
removed.

Reply:
I meant it is OK to keep it as is, and have an issue to track the test update. Ofc removing is also fine, we'll add the test back by the time we get #103 done. Just didn't want you to sweat.